### PR TITLE
[SMD-67] TF Round 4 validation - modify programme risk validation con…

### PR DESCRIPTION
**Pull request by cyrusdobbs**
…dition

https://dluhcdigital.atlassian.net/browse/SMD-67

### Change description
The requirement changed from the validation requiring 3 programme level risks --> 1 programme level risk

- [X] Unit tests and other appropriate tests added or updated
- [X] README and other documentation has been updated / added (if needed)
- [X] Commit messages are meaningful and follow good commit message guidelines (e.g. "FS-XXXX: Add margin to nav items preventing overlapping of logo")

### How to test
_If manual testing is needed, give suggested testing steps_

### CODE REVIEW

File: core/validation/specific_validations/towns_fund_round_four.py [hunk 1]
```diff
@@ -57,15 +57,15 @@ def validate_programme_risks(workbook: dict[str, pd.DataFrame]) -> list["TownsFu
...
-    # TODO: Confirm if 1 or 3 programme risks are required and change this function accordingly
-    if len(risk_programme_ids) < 3:
+    if len(risk_programme_ids) < 1:
         return [
             TownsFundRoundFourValidationFailure(
                 tab="Risk Register",
                 section="Programme Risks",
-                message="You have not entered enough programme level risks. You must enter 3 programme level risks",
+                message="You have not entered enough programme level risks. "
+                "You must enter at least 1 programme level risks",
```
Comment (Joel-Pearce) [inline core/validation/specific_validations/towns_fund_round_four.py:68]: should be "1 programme level risk" as not plural

**Review by Joel-Pearce — CHANGES_REQUESTED** [review]

File: core/validation/specific_validations/towns_fund_round_four.py [hunk 2]
```diff
@@ -57,15 +57,15 @@ def validate_programme_risks(workbook: dict[str, pd.DataFrame]) -> list["TownsFu
                      DataFrames representing each sheet in the Round 4 submission.
     :return: ValidationErrors
     """
-    risk_programme_ids = list(workbook["RiskRegister"]["Programme ID"].dropna())
+    risk_programme_ids = workbook["RiskRegister"]["Programme ID"].dropna()
 
-    # TODO: Confirm if 1 or 3 programme risks are required and change this function accordingly
-    if len(risk_programme_ids) < 3:
+    if len(risk_programme_ids) < 1:
```
Comment (bobbi-hbrown) [inline core/validation/specific_validations/towns_fund_round_four.py:62]: As the options are only 1 or 0 now, using `if not risk_programme_ids` is slightly more efficient

Comment (cyrusdobbs) [reply core/validation/specific_validations/towns_fund_round_four.py:62]: `risk_programme_ids` is a series so it won't evaluate to a boolean like this

**Review by Joel-Pearce — APPROVED** [review]
LGTM

**Pull request merged**